src/Admin/CitizenInitiativeAskingHelpAdmin.php

```diff
@@ -25,13 +26,24 @@ class CitizenInitiativeAskingHelpAdmin extends AbstractAdmin
 
     private $skillRepository;
 
+    /**
+     * CitizenInitiativeAskingHelpAdmin constructor.
```

Comment:
Docblock is not needed, constructor arguments must be typehinting instead.

Reply:
Here, the parameters are no typed and put without explaination